subsys/bluetooth/controller/ll_sw/ull_sync_internal.h

```diff
@@ -11,3 +11,4 @@ void ull_sync_release(struct ll_sync_set *sync);
 void ull_sync_setup(struct ll_scan_set *scan, struct ll_scan_aux_set *aux,
 		    struct node_rx_hdr *node_rx, struct pdu_adv_sync_info *si);
 void ull_sync_done(struct node_rx_event_done *done);
+struct ll_sync_set *ull_sync_is_enabled_get(uint16_t handle);
```

Comment:
Move it to after the declaration of `ull_sync_handle_get`

Reply:
Done